modules/kubernetes-addons/aws-load-balancer-controller/values.yaml

```diff
@@ -3,3 +3,5 @@ region: ${aws_region}
 serviceAccount:
   create: false
   name: ${service_account_name}
+image:
+  repository: ${ecr_registry}/amazon/aws-load-balancer-controller
```

Comment:
> When you deploy add-ons such as the Installing the AWS Load Balancer Controller add-on, the [Amazon VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html#updating-vpc-cni-add-on), [kube-proxy](https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html#updating-kube-proxy-add-on), or [storage drivers](https://docs.aws.amazon.com/eks/latest/userguide/storage.html) to your cluster, your nodes might pull the container image from an Amazon EKS Amazon ECR private repositoryWhen you deploy add-ons such as the Installing the AWS Load Balancer Controller add-on, the [Amazon VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html#updating-vpc-cni-add-on), [kube-proxy](https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html#updating-kube-proxy-add-on), or [storage drivers](https://docs.aws.amazon.com/eks/latest/userguide/storage.html) to your cluster, your nodes might pull the container image from an Amazon EKS Amazon ECR private repository

Should we do a similar change for the EFS CSI driver?

Reply:
Great catch - The [default helm chart values](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/charts/aws-efs-csi-driver/values.yaml) for that addon points to a public ECR registry, so I don't think we will run into the same problems with the EFS CSI driver. I think the VPC CNI plugin will need similar updates, and in that case we may want to move the registry mapping local up a level so it can be referenced by more than one addon.